base/sparse/cholmod.jl

```diff
@@ -1330,7 +1338,7 @@ function cholfact!(F::Factor{Tv}, A::Sparse{Tv}; shift::Real=0.0) where Tv
     cm = common()
```

Comment:
Is it really necessary to have a function `common()` to get the global variable  `commonStruct`?

Reply:
No, I don't think so. It used to take an argument but that was broken and got removed so I think we can just delete the function.

Reply:
done

Reply:
@andreasnoack would you be opposed to renaming `commonStruct` to `common_struct` ? The cammelcase doesn't seem very commonly used in julia code.